ember-cli-build.js

```diff
@@ -178,5 +178,10 @@ function getEmberCLIBlueprints() {
   fileMap['index.html'] = fs.readFileSync('blueprints/index.html').toString();
   fileMap['test-start-app'] = fs.readFileSync('blueprints/start-app.js').toString();
 
+  // use custom model blueprint since the one currently defined in
+  // ember-cli-legacy-blueprints contains custom logic for import statements in
+  // the blueprints' index.js, which is not (yet) considered in ember-twiddle
```

Comment:
Please also remove line for `model` in `legacyBlueprintFiles` hash. (This is overwriting it.)